Makefile

```diff
@@ -263,6 +263,9 @@ devenv-mysql:
 protobuf: ## Compile protobuf definitions
 	bash scripts/protobuf-check.sh
 	bash pkg/plugins/backendplugin/pluginextensionv2/generate.sh
+	bash pkg/plugins/backendplugin/secretsmanagerplugin/generate.sh
+	bash pkg/services/store/entity/generate.sh
+	bash pkg/infra/grn/generate.sh
```

Comment:
Shall we maybe have a script that `find`'s all `.proto` files and applies `protoc` command to them?

Reply:
There is some nuance to the generate.sh files right now, but we could potentially do something like that if we can resolve the differences that we need currently. That sounds like a separate PR though ;)